src/components/Accordion/AccordionHeader.tsx

```diff
@@ -4,25 +4,25 @@ import React, { ReactElement, cloneElement, isValidElement, useState } from 'rea
 import { IconSize } from '@foundation/Icon/IconSize';
 import { IconProps } from '@foundation/Icon/IconProps';
 import { merge } from '@utilities/merge';
-import { AccordionHeaderIconSize, AccordionHeaderProps } from './types';
+import { AccordionHeaderProps } from './types';
 import { AccordionHeaderIcon } from './AccordionHeaderIcon';
 
+export const ACCORDION_HEADER_TEST_ID = 'fondue-accordion-header';
+
 export const AccordionHeader = ({
     isOpen = false,
     decorator,
     disabled = false,
     children,
-    size = AccordionHeaderIconSize.Medium,
-    bold = true,
-    type,
+    'data-test-id': testId = ACCORDION_HEADER_TEST_ID,
 }: AccordionHeaderProps) => {
     const [headingRef, setHeadingRef] = useState<HTMLSpanElement | null>(null);
-    const icon = <AccordionHeaderIcon isOpen={isOpen} disabled={disabled} size={size} type={type} />;
+    const icon = <AccordionHeaderIcon isOpen={isOpen} disabled={disabled} />;
 
     return (
-        <span data-test-id="accordion-header" className="tw-block tw-px-8 tw-py-6">
+        <span data-test-id={testId} className="tw-block tw-px-8 tw-py-6">
```

Comment:
Please provide a default with the fondue prefix.

Reply:
The default value comes from the `ACCORDION_HEADER_TEST_ID` constant value `fondue-accordion-header` on line 10 :)